plugins/outputs/wavefront/sample.conf

```diff
@@ -67,4 +69,29 @@
   # insecure_skip_verify = false
 
   ## HTTP Timeout
-  #timeout="10s"
\ No newline at end of file
+  #timeout="10s"
+
+  ## Authentication for Direct Ingestion.
+  ## Direct Ingestion requires one of: `token`,`auth_csp_api_token`, or `auth_csp_client_credentials`
+  ## See https://docs.wavefront.com/csp_getting_started.html to learn more about using CSP credentials with Wavefront.
+  ## Not required if using a Wavefront proxy.
+
+  ## Wavefront API Token Authentication. Ignored if using a Wavefront proxy.
+  # 1. Click the gear icon at the top right in the Wavefront UI.
+  # 2. Click your account name (usually your email)
+  # 3. Click *API access*.
+  #token = "YOUR_TOKEN"
+
+  # Optional. defaults to "https://console.cloud.vmware.com/"
+  # Ignored if using a Wavefront proxy or a Wavefront API token.
+  #auth_csp_base_url=https://console.cloud.vmware.com
```

Comment:
```suggestion
  ## Optional. defaults to "https://console.cloud.vmware.com/"
  ## Ignored if using a Wavefront proxy or a Wavefront API token.
  # auth_csp_base_url=https://console.cloud.vmware.com
```

Reply:
do you happen to have a linter that you use to spot the whitespace issues in comments?